src/libraries/System.Text.RegularExpressions/gen/Resources/xlf/Strings.de.xlf

```diff
@@ -228,8 +228,8 @@
         <note />
       </trans-unit>
       <trans-unit id="RegexMethodMustHaveValidSignatureMessage">
-        <source>Regex method must be partial, parameterless, non-generic, and return Regex</source>
-        <target state="translated">Die Regex-Methode muss partiell, parameterlos, nicht generisch sein und Regex zurückgeben.</target>
+        <source>RegexGenerator method must be partial, parameterless, non-generic, non-abstract, and return Regex</source>
+        <target state="needs-review-translation">Die Regex-Methode muss partiell, parameterlos, nicht generisch sein und Regex zurückgeben.</target>
```

Comment:
```suggestion
        <target state="translated">Die Regex-Methode muss partiell, parameterlos, nicht generisch sein und Regex zurückgeben.</target>
```
As a native German speaker, I can verify that the translation is fine.

Reply:
thanks @deeprobin . While I've zero doubt you're correct, I don't think the localization process expects us to localize in the repo. the strings get sucked into their side, and come back translated. More generally, we have no way of reviewing these (depending on the language!)

So I'll reject this suggestion unfortunately